addons/knowledge/data/knowledge_data.xml

```diff
@@ -145,8 +145,9 @@
             </field>
             <field name="parent_id" eval="False" />
             <field name="internal_permission">write</field>
+            <field name="favorite_ids" eval="[(5,0), (0, 0, {'sequence': 0, 'user_id': ref('base.user_admin')})]"/>
```

Comment:
Same remark: to move to demo if necessary, as standard data should not bloat your db. Demo data are there for that purpose.